### ISSUE TITLE
Ability to set selinux contexts for volumes

### ISSUE THREAD

**Issue by S-B-V** [cockpit-project/cockpit-podman#362]
SELinux will deny it and it is not possible to use rw:z or ro:z and there for SELinux needs to be deactivated permanently.
If I create a container with permissive SELinux it will work until reboot that SELinux is back and will destroy the container. Even after deactivating SELinux it is not possible to the container again.
I am using FCOS for testing FCOS with cockpit and podman but I may stay longer with RancherOS until it is really usable (cockpit-podman).

**Comment by marusak** [comment]
I am not sure if I understand fully, but

>  it is not possible to use rw:z or ro:z

Indeed, it is not yet possible to set selinux context on volumes from cockpit-podman. It is tracked in https://bugzilla.redhat.com/show_bug.cgi?id=1811773

> If I create a container with permissive SELinux it will work until reboot that SELinux is back and will destroy the container. Even after deactivating SELinux it is not possible to the container again.

You can disable selinux permanently so it does not start after reboot. If the container gets corrupted because selinux was present and it is not possible to start it back up, this might be issue in podman itself. You may consider opening bug in https://github.com/containers/libpod

**Comment by S-B-V** [comment]
Do not get me wrong, I disabled SELinux but this is just a bad workaround. I also do not care that SELinux destroyed my container as it is working as expecting. The problem is the missing z option. I will not create a bug report if it is just related to this issue.
I will test if everything works as expected with SELinux enabled and :z option by doing it manual via CLI as soon as I have time for that. This issue which I opened here is just related to cockpit-podman and the missing feature.

I also appreciate all the hard work and I like it already but I would like to have it awesome and as I am not a coder at least to provide some feedback to improve the product and usability.

I will keep track on the bugzilla ID, many thanks for the ID
